Automattic/woocommerce-payments
# [Simplifying Deposits UI] Next deposits table heading

Fixes #5695 

#### Changes proposed in this Pull Request









The change in this PR is small however lays a bit of a foundation for subsequent issues, particularly https://github.com/Automattic/woocommerce-payments/issues/5699.

In this PR I've created a Next Deposits details child component which we'll use to house all of the next deposit section (heading, table, notice).

![](https://d.pr/i/NM6Rfs+)

I haven't added unit tests for this PR as there's nothing that needs to be tested apart from the rendering of the component (which seems pointless at this stage because the snapshots will be changing a lot and will constantly be failing tests). We will add tests once there's more functionality and more UI elements added.

#### Testing instructions





1. Pull the changes and run `npm run build:deps`
2. Set `_wcpay_feature_simplify_deposits_ui` to `1` in wp-options table
3. From WP Admin go to Payments to view the Overview page
4. Check the Next deposits heading loads as per designs

-------------------


**Post merge**
